### ISSUE TITLE
When adding a DwCA as a source skip eml.xml and meta.xml files

### ISSUE THREAD

**Issue by mike-podolskiy90** [gbif/ipt#2630]
They should be skipped as they cannot be the source of the dataset

**Comment by MathewBiddle** [comment]
The eml.xml and meta.xml should be skipped when **updating** a dataset in the IPT (ie. updating the data files only). If you **start** a dataset with a DwCA .zip package that contains meta.xml and eml.xml, the IPT knows what to do with them (adds the eml.xml metadata to the Metadata and adds the meta.xml mappings to the Mappings and only adds the data files to Source Data). It just doesn't understand when you add the DwCA .zip as "Source Data".

I've ran into circumstances where I'd like to update the entire dataset by providing a DwCA .zip file. This includes updating the mapping in meta.xml and metadata in eml.xml. Could the IPT decipher this nuance when uploading a package via the Source Data "+Add" option? The metadata section allows one to update the metadata from an eml.xml file, but I can't figure out how to submit a mapping meta.xml file if there are changes. Unfortunately this means I have to edit the mappings by hand, which takes forever. 

FYI @sformel @jdpye

**Comment by mike-podolskiy90** [comment]
Thank you for the suggestion @MathewBiddle, I'll keep that in mind

**Comment by jdpye** [comment]
I agree with Mat. Updating a dataset with a package that includes an eml.xml file, and using that eml.xml to update the metadata, would be a very fine feature to have for folks who are autogenerating darwin core archives w/ metadata included, and updating that metadata from an institutional repository. It would be incredibly useful to be able to read my meta.xml file in to override mappings for cores and extensions.

 I'm currently doing this work via Requests in Python, and taking an extra step to update the metadata with a bare eml.xml upload after already having built the full archive, with meta.xml having updated mappings and eml.xml having updated metadata inside.
